README.md

```diff
@@ -0,0 +1,197 @@
+# Modbus communication library
+
+This library provides functions to send and receive messages over serial line
+to/from other devices. Runs only on FPGA. Used mainly to communicate on a
+master-slave bus, usually with daisy chained addressing of slave modules.
+
+**[CRC16](https://crccalc.com) calculation is not included**. Calling process should calculate and
+include proper CRC16 as data payload, and check received data CRC16 (or any
+other CRC).
+
+## Physical interface
+
+Serial interface is constructed from two DIO pins. Obviously one pin has to be
+configured for input (receiving), the other for output (transmitting). Usually
+NI-9401 module, providing total 8 DIOs, with split for 4 in and 4 output ports,
+is used.
+
+## LabView/Host interface
+
+[Common_FPGA_HealthAndStatus
+library](https://github.com/lsst-ts/Common_FPGA_HealthAndStatus/) is used for
+health and status FIFO. That provides datatype for the FIFO.
```

Comment:
Please consider "is a datatype for a health and status FIFO" or something along those lines.